Tasks/InstallSSHKeyV0/task.json

```diff
@@ -29,11 +29,11 @@
             "aliases": [
                 "knownHostsEntry"
             ],
-            "type": "string",
+            "type": "multiLine",
             "label": "Known Hosts Entry",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "The entry for this SSH key for the known_hosts file."
+            "helpMarkDown": "The entry for this SSH key for the known_hosts file. Supports several hosts."
```

Comment:
Nit: After making changes like this, make sure you build the task and include the updated .resjson files - these contain changes used for localization.

Reply:
Updated .resjson file (en-US)